lib/daru/index.rb

```diff
@@ -330,5 +341,11 @@ def values
     def inspect
       "Daru::MultiIndex:#{self.object_id} (levels: #{levels}\nlabels: #{labels})"
     end
+
+    def factor_out
+      # Remove levels not needed for display
+      useless = levels.index { |l| l.length > 1 }
+      drop_left_level useless
+    end
```

Comment:
@v0dro No, they don't follow the same convention. I think there's a bug in our implementation.

```python
>>> index = pd.MultiIndex.from_tuples([['a', 'one'], ['b', 'two']], names=['first', 'second'])
>>> s = pd.Series(np.random.randn(8), index=index)
>>> s
first  second
a      one       1.387915
b      two      -1.169805
dtype: float64
>>> s['a']
second
one    1.387915
dtype: float64
```